docker/base/os/alpine/Dockerfile.alpine

```diff
@@ -1,8 +1,8 @@
 # This is the dockerfile for generating base images for Alpine
 # contains the minimal dependencies needed for V to run
-FROM alpine:3.11.6
+FROM alpine:3.17
```

Comment:
If you want to keep up with the latest, just remove the version number.  Whenever the image is built, it will use `alpine:latest`.